Compilation on MSVS 2019 dealii.master

Hi, I'm try deal.ii for the first time. I have MSVS 2019 and I have managed to run cmake as instructed. However I get compilation errors. I know its experiemental but would appreciate some help.

C:\dealii\include\deal.II/base/symmetric_tensor.h(2428,1): error C2668: 'dealii::internal::SymmetricTensorImplementation::unrolled_to_component_indices': ambiguous call to overloaded function

C:\dealii\include\deal.II/base/tensor_product_polynomials.h(482,16): error C2466: cannot allocate an array of constant size 0

Error	C2133	'indices': unknown size	obj_base_release	C:\dealii\include\deal.II\base\tensor_product_polynomials.h	482

Error	C3615	constexpr function 'dealii::Tensor<1,6,Number>::Tensor' cannot result in a constant expression
        with
        [
            Number=double
        ]	obj_physics_elasticity_release	C:\dealii\include\deal.II\physics\elasticity\standard_tensors.h	72	

Error	C3615	constexpr function 'dealii::Tensor<1,6,Number>::Tensor' cannot result in a constant expression
        with
        [
            Number=double
        ]	obj_physics_elasticity_release	C:\dealii\include\deal.II\physics\elasticity\standard_tensors.h	121

I only get 2 errors now they are both:

58>'rm' is not recognized as an internal or external command,
58>operable program or batch file.
58>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(230,5): error MSB6006: "cmd.exe" exited with code 9009.
58>Done building project "kill-step.release-OK.vcxproj" -- FAILED.

the other the debug

PS I'm working on the master now. 9.1.1 built with only 2 errors:

58>'rm' is not recognized as an internal or external command,
58>operable program or batch file.
58>C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(230,5): error MSB6006: "cmd.exe" exited with code 9009.
58>Done building project "kill-step.release-OK.vcxproj" -- FAILED.
the other is the debug

The master still has:

Severity	Code	Description	Project	File	Line	Suppression State
Error (active)	E2661	variable in constexpr function does not have automatic storage duration	obj_physics_elasticity_debug	C:\dealii\include\deal.II\base\table_indices.h	351	

this is related to:
template <int N>
DEAL_II_CONSTEXPR inline std::size_t &TableIndices<N>::
                                      operator[](const unsigned int i)
{
  AssertIndexRange(i, N);
  return indices[i];
}

and many:

Severity	Code	Description	Project	File	Line	Suppression State
Error	C3615	constexpr function 'dealii::Tensor<1,1,double>::Tensor' cannot result in a constant expression	obj_physics_elasticity_debug	C:\dealii\include\deal.II\physics\elasticity\standard_tensors.h	72

this is related to:
#ifdef DEAL_II_HAVE_CXX14_CONSTEXPR_CAN_CALL_NONCONSTEXPR
= unit_symmetric_tensor()
#endif

More specific error

27>kinematics.cc
26>kinematics.cc
27>C:\dealii\include\deal.II/physics/elasticity/standard_tensors.h(72,37): error C3615: constexpr function 'dealii::Tensor<1,1,double>::Tensor' cannot result in a constant expression
26>C:\dealii\include\deal.II/physics/elasticity/standard_tensors.h(72,37): error C3615: constexpr function 'dealii::Tensor<1,1,double>::Tensor' cannot result in a constant expression
27>C:\dealii\include\deal.II/base/tensor.h(466,3): message : failure was caused by 'constexpr' constructor not initializing member 'dealii::Tensor<1,1,double>::values'
27>C:\dealii\include\deal.II/physics/elasticity/kinematics.h(283): message : see reference to class template instantiation 'dealii::Physics::Elasticity::StandardTensors<1>' being compiled
27>C:\dealii\build\source\physics\elasticity\kinematics.inst(18): message : see reference to function template instantiation 'dealii::Tensor<2,1,double> dealii::Physics::Elasticity::Kinematics::F<1,double>(const dealii::Tensor<2,1,double> &)' being compiled
27>C:\dealii\include\deal.II/physics/elasticity/standard_tensors.h(121,24): error C3615: constexpr function 'dealii::Tensor<1,1,double>::Tensor' cannot result in a constant expression
27>C:\dealii\include\deal.II/base/tensor.h(466,3): message : failure was caused by 'constexpr' constructor not initializing member 'dealii::Tensor<1,1,double>::values'
27>C:\dealii\include\deal.II/physics/elasticity/standard_tensors.h(72,37): error C3615: constexpr function 'dealii::Tensor<1,3,Number>::Tensor' cannot result in a constant expression
27>C:\dealii\include\deal.II/physics/elasticity/standard_tensors.h(72,37): error C3615:         with
27>C:\dealii\include\deal.II/physics/elasticity/standard_tensors.h(72,37): error C3615:         [
27>C:\dealii\include\deal.II/physics/elasticity/standard_tensors.h(72,37): error C3615:             Number=double
27>C:\dealii\include\deal.II/physics/elasticity/standard_tensors.h(72,37): error C3615:         ]

--- a/include/deal.II/base/symmetric_tensor.h
+++ b/include/deal.II/base/symmetric_tensor.h
@@ -2401,9 +2401,10 @@ namespace internal
     // this function is for tensors of a rank not already handled
     // above
     template <int dim, int rank_>
-    DEAL_II_CONSTEXPR inline TableIndices<rank_>
-    unrolled_to_component_indices(const unsigned int i,
-                                  const std::integral_constant<int, rank_> &)
+    DEAL_II_CONSTEXPR inline
+      typename std::enable_if<rank_ != 2, TableIndices<rank_>>::type
+      unrolled_to_component_indices(const unsigned int i,
+                                    const std::integral_constant<int, rank_> &)
     {
       (void)i;
       Assert(

